### DISCUSSION TITLE
Problem with BYD Seal U and Protocol AutoPID > Burglar Alarm

### DISCUSSION THREAD

**Discussion by cbeham1** [meatpiHQ/wican-fw#290] [Q&A]
if i use the protocol AUTOPID the car always start the burglar alarm (light & sound) after locking it.
if i change back do slcan protocol there is no more information on mqtt but no alarm anyway too.

BYD SEAL U 12/2024 Version 1.2.0

**Comment by meatpiHQ** [comment]
@cbeham1 did you try to leave the car unlocked?

**Reply by T1983M** [reply]
I can confirm the alarm behaviour with the BYD Dolphin. My solution was to basicly sit in the car with the ignition on to do the initial readings on the PIDs via Carscanner

**Reply by T1983M** [reply]
Hej @meatpiHQ I looked deeper into the Car alarm thing. If the System is in AutoPid and you lock the car you trigger the alarm after 5 seconds (which is the polling rate). I set up the Sleep timer with a fitting voltage and reduced the sleep time to the minimum (1min) if i turn the ignition off and lock the car after more than 1 minute the alarm does not go off.

Therefore, would it be possible to reduce the sleep time to seconds? I would like to set it down to 15 seconds. That would be enough, since it would ping the current data when i arrive, and i can not get out of the car and lock it in under 15seconds ;)

**Reply by meatpiHQ** [reply]
Yes it's possible, but combined with the wrong voltage that could trigger the device never waking up. I think it's better to add an option to stop polling, if voltage drops under sleep voltage.

**Reply by T1983M** [reply]
That sounds great..... this would give even more felxibility. 

A thing that i am currently investigating is, on how the battery current behaves if the vehicle is charging. That could cause the battery voltage to go up and wake up the Auto Polling. Could it be a possibbilty to passively listen to the bus, and just MQTT send the most recent value of interest on a regular basis.

**Reply by meatpiHQ** [reply]
It's possible to listen to the bus and update the values of the car support CAN frames broadcasting while it's charging and we know which frames contain the values. Most cars support OBD protocol which means WiCAN has to send a request frame to get a response.

**Comment by cbeham1** [comment]
Hi. Yes. But a little bit later the car locks it self.
BUT the Major Problem is, that it does NOT Work with the Atto3 Profile for Seal U :(
Also. With the really cool new Version 4 i get nothing...
Greetings chris

**Reply by meatpiHQ** [reply]
@cbeham1 Did you try to use car scanner app? 

I can create a new profile for the Seal U with your help.

**Reply by dasmith83** [reply]
@meatpiHQ  - Would you be able to help setup a BYD Seal profile if I'm able to grab the logs from car scanner? I'll give the guide below a go later today :) Thanks in advance

**Comment by cbeham1** [comment]
Would be very very nice :)
THANK YOU

Please tell me what i have to do (dummy like)

Chris

**Reply by meatpiHQ** [reply]
@cbeham1 I've added some documentation on how to connect car scanner to WiCAN and export the logs. Let me know if you have any issues.

https://meatpihq.github.io/wican-fw/car_scanner/usage

**Comment by jcharwell21** [comment]
I received my WICAN Pro this week. I am using with a Seal U Design. I experienced this behaviour once. I opened and relocked the car and I did not happen again. Is this problem still open? Is there a fix or workaround? Is there any information that I can pull off the setup that would help?

**Reply by meatpiHQ** [reply]
@jcharwell21 You must update your Seal, see this comment.

https://github.com/meatpiHQ/wican-fw/discussions/122#discussioncomment-13272477
